certbot-apache/tests/util.py

```diff
@@ -69,8 +70,7 @@ def setUp(self, test_dir="debian_apache_2_4/multiple_vhosts",
               vhost_root="debian_apache_2_4/multiple_vhosts/apache2/sites-available"):
         super().setUp(test_dir, config_root, vhost_root)
 
-        zope.component.provideUtility(display_util.FileDisplay(sys.stdout,
-                                                               False))
+        display_obj.set_display(display_util.FileDisplay(sys.stdout, False))
```

Comment:
I think using this here is a problem because we're using the internal function `certbot._internal.display.obj.set_display` across a package boundary. I think this can cause our tests to start failing before a major version bump which isn't supposed to happen.

Luckily, I think we can easily avoid this with this patch: https://gist.github.com/bbfc65284e60ae577f6929d5382f6150

Reply:
Indeed!